launch-script.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
```

Comment:
You need to set `TMPDIR`, Firefox needs it. I didn't say to remove it completely.